src/ducks/apps/components/ApplicationRouting/ChannelRoute.jsx

```diff
@@ -41,7 +41,7 @@ export const ChannelRoute = ({
           isAppFetching={isAppFetching}
           isInstalling={isInstalling}
           onCurrentChannel={() => redirectTo(appPath)}
-          onNotInstalled={() => redirectTo(appPath)}
+          onNotHandled={() => redirectTo(appPath)}
```

Comment:
What does `onNotHandled` mean ?

Reply:
For the case where the channel modal doesn't not handle this application according to the app state/status

Reply:
It seems that we should not even get there, should we ?

Reply:
Since we don't check those statuses in the route actually we can get there